UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/HealthCheck/EventHubLoggingHealthCheck.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using System.Threading;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.Common.Logging;
+
+namespace UKHO.ExchangeSetService.Common.HealthCheck
+{
+    public class EventHubLoggingHealthCheck : IHealthCheck
+    {
+        private readonly IEventHubLoggingHealthClient eventHubLoggingHealthClient;
+        private readonly ILogger<EventHubLoggingHealthCheck> logger;
+
+        public EventHubLoggingHealthCheck(IEventHubLoggingHealthClient eventHubLoggingHealthClient, ILogger<EventHubLoggingHealthCheck> logger)
+        {
+            this.eventHubLoggingHealthClient = eventHubLoggingHealthClient;
+            this.logger = logger;
+        }
+
+        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default)
+        {
+            var healthCheckResult = await eventHubLoggingHealthClient.CheckHealthAsync(context);
+            if (healthCheckResult.Status == HealthStatus.Healthy)
+            {
+                logger.LogInformation(EventIds.EventHubLoggingIsHealthy.ToEventId(), $"Event hub is healthy");
```

Comment:
Could this health log be at a different level please (e.g. Debug):
```suggestion
                logger.LogDebug(EventIds.EventHubLoggingIsHealthy.ToEventId(), $"Event hub is healthy");
```

Reply:
Fixed